README.md

```diff
@@ -576,31 +577,41 @@ Here the steps to compute the lower/upper bounds for the table query:
 Now that we have the lower and upper bound keys, simply perform your query:
 
 ```
-eosc get table eosforumrcpp eosforumrcpp vote --index 3 --key-type i128 --lower-bound 0x000000000000000090b1ca57619db1ca --upper-bound 0xffffffffffffffff90b1ca57619db1ca
+eosc get table eosio.forum eosio.forum vote --index 3 --key-type i128 --lower-bound 0x000000000000000090b1ca57619db1ca --upper-bound 0xffffffffffffffff90b1ca57619db1ca
 ```
 
 You will see only the proposals that voter `testusertest` voted for.
 
-#### JSON Structure Guidelines
+#### Proposal JSON Structure Guidelines
```

Comment:
@Novusphere Should fixed the issue your raised a month ago. Feel free to review to improve this addition.